charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py

```diff
@@ -207,10 +207,17 @@ def patch_json_set_editable_as_variable(content):
 
 
 def jsonnet_import_callback(base, rel):
+    # rel_base is the path relative to the current cwd.
+    # see https://github.com/prometheus-community/helm-charts/issues/5283
+    # for more details.
+    rel_base = base
+    if rel_base.startswith(os.getcwd()):
+        rel_base = base[len(os.getcwd())+1:]
```

Comment:
https://docs.python.org/3.9/library/stdtypes.html?highlight=removeprefix#str.removeprefix
`removeprefix` is a thing - should we assume everyone is using python 3.9+?

Reply:
I also don't know why the `+1` is there, I will get rid of it.

Reply:
Feel like `removeprefix` is too small to raise the min version here.

Reply:
Alright, then I would leave it at that.